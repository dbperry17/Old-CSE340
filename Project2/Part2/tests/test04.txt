
(cse340) + > <=

$ language 43

&&
