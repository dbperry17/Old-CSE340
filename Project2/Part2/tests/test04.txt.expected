4 NUM 43
4 ID language
2 ID cse340
